### DISCUSSION TITLE
Config: Unknown transform: HttpMethod;Set

### DISCUSSION THREAD

**Discussion by pcfco-kmason** [dotnet/yarp#1262] [Q&A] [answered]
Hi - I'm sure this is a misunderstanding on the config setup, but is this a valid transform? 

"Transforms": [          
          {
            "HttpMethod": "POST",
            "Set": "GET"
          }                        
        ]

reference: https://microsoft.github.io/reverse-proxy/articles/transforms.html#httpmethod

I'm getting the below when inserting the above. If I just use something like { "PathRemovePrefix": "/api" } it's fine.

System.InvalidOperationException
  HResult=0x80131509
  Message=Unable to load or apply the proxy configuration.
  Source=Yarp.ReverseProxy
...
Inner Exception 1:
AggregateException: The proxy config is invalid. (Unknown transform: HttpMethod;Set)

Inner Exception 2:
ArgumentException: Unknown transform: HttpMethod;Set

Thank you!

**Comment by Kahbazi** [comment, marked as answer]
@pcfco-kmason This looks like a problem with docs. Try `HttpMethodChange` instead of `HttpMethod`.
```json
"Transforms": [
{
"HttpMethodChange": "POST",
"Set": "GET"
}
]
```

**Reply by pcfco-kmason** [reply]
@Kahbazi Thanks for your help - that fixed it! Much appreciation.

**Reply by Tratcher** [reply]
Fixed by https://github.com/microsoft/reverse-proxy/pull/1264
